torch/csrc/jit/export_module.cpp

```diff
@@ -24,7 +24,86 @@ namespace {
 ExportModuleExtraFilesHook& GetExtraFilesHook() {
   static ExportModuleExtraFilesHook func = nullptr;
   return func;
-};
+}
+
+static IValue Tup(std::vector<IValue> ivalues) {
```

Comment:
Would it make sense to have a version of this that accepts rvalue ref? I see call-sites for this where passed in vector is not used afterwards and we still end up copying it during the call to this function.

Reply:
I assume Tup() is a simple helper function. Currently the vector is not used later but not guaranteed. In addition, it's in the export process where I don't think the perf saving (if there is any) with rvalue ref would be noticeable. To keep it simple I'd prefer one API for the helper function.